cmake/qr-code-generator.cmake

```diff
@@ -0,0 +1,18 @@
+CPMAddPackage(
+  NAME qr-code-generator
+  GITHUB_REPOSITORY nayuki/QR-Code-generator
+  GIT_TAG 720f62bddb7226106071d4728c292cb1df519ceb
+  LANGUAGES CXX
```

Comment:
Even if you're not going to build this, or it doesn't come with a cmake file, you should still mark this as `DOWNLOAD_ONLY`

Reply:
Added